vnext/PropertySheets/Bundle.props

```diff
@@ -6,10 +6,49 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
   <PropertyGroup>
+
+    <!-- 
+      UseBundle causes the bundle to be created and included inside the application install.  And defines the BUNDLE #define in code,
+      which causes the app to load from the bundle file rather than loading the bundle from a bundle server
+    -->
     <UseBundle Condition=" '$(Configuration)' == 'Debug' and '$(UseBundle)' == '' ">false</UseBundle>
     <UseBundle Condition=" '$(Configuration)' == 'Release' and '$(UseBundle)' == '' ">true</UseBundle>
-    <BundleContentRoot Condition=" '$(BundleContentRoot)' == '' ">Bundle</BundleContentRoot>
-    <BundleContent Condition = " '$(BundleContent)' == '' and '$(BundleContentRoot)' != '' ">$(BundleContentRoot)\**\*</BundleContent>
+
+    <!-- Root directory where bundle assets will be copied to, be sure to update BundleContent if you change this -->
+    <BundleContentRoot Condition=" '$(BundleContentRoot)' == '' ">$([MSBuild]::NormalizePath('$(ProjectDir)\Bundle'))</BundleContentRoot>
```

Comment:
I *think* ProjectDir is a VS thing whereas MSBuildProjectDirectory is the MSBuild property - are you seeing this work on the command line too?